utt/utt-client-api/src/User.cpp

```diff
@@ -96,6 +96,8 @@ struct User::Impl {
   std::vector<libutt::api::Coin> coins_;         // User's unspent UTT coins (tokens)
   std::optional<libutt::api::Coin> budgetCoin_;  // User's current UTT budget coin (token)
   std::set<std::string> budgetNullifiers_;
```

Comment:
this set ```budgetNullifiers_``` should also be persisted no?

Reply:
no, it is stored from the current budget coin storage. We don't really care about the old budget coins (once we have a persistent storage that saves the latest one)